packages/devextreme/testing/tests/DevExpress.ui.widgets/treeViewParts/focusing.js

```diff
@@ -7,6 +7,7 @@ import TreeViewTestWrapper from '../../../helpers/TreeViewTestHelper.js';
 import keyboardMock from '../../../helpers/keyboardMock.js';
 import { shouldSkipOnMobile } from '../../../helpers/device.js';
 
+const TREEVIEW_CLASS_PREFIX = 'dx-treeview';
```

Comment:
Export the corresponding classes from the files that contain the components where they are used